### PR TITLE
[#4] [Chore] [iOS] Configure CD for the project

### LINKED ISSUE #4: [Chore] [iOS] Configure CD for the project
## Why

To improve production quality, continuous deployment (CD) plays an important part in the development process.

## Acceptance criteria

Set up a CD pipeline that satisfies the development standard.

### PR DESCRIPTION
**Pull request by nmint8m**
- Close #4

## What happened 👀

In this PR, I created the CD pipeline for the iOS project. It support deploy the production app to the Firebase.

## Insight 📝

As the iOS in this IC uses the beta version of Swift, so we should use the Xcode version 15.0-beta and the iOS version 17 beta on MacOS 13 to build archive the app. The point is that Apple prevents summit app with beta version, so it rejects the build. Currently, we can only submit the production build (with adhog configuration) onto Firebase.

## Proof Of Work 📹

The CD pipeline successfully runs.

image

### CODE REVIEW

File: .github/workflows/ios_deploy_production_build.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+name: iOS - Deploy Production Builds To App Store
+
+on:
+  push:
+    branches:
+      - main
+      - chore/4-cd-ios
```
Comment (markgravity) [inline .github/workflows/ios_deploy_production_build.yml:7]: Debug code detected 🤓

File: buildSrc/src/main/kotlin/appPackage/BuildKonfig.kt.example [hunk 2]
```diff
@@ -1,9 +1,9 @@
-object BuildKonfig {
-    const val CLIENT_ID_PRODUCTION = ""
-    const val CLIENT_SECRET_PRODUCTION = ""
-    const val BASE_URL_PRODUCTION = ""
-
-    const val CLIENT_ID_STAGING = ""
-    const val CLIENT_SECRET_STAGING = ""
-    const val BASE_URL_STAGING = ""
-}
+//object BuildKonfig {
```
Comment (markgravity) [inline buildSrc/src/main/kotlin/appPackage/BuildKonfig.kt.example:1]: Please help uncomment them 🙏

Comment (nmint8m) [reply buildSrc/src/main/kotlin/appPackage/BuildKonfig.kt.example]: @markgravity Fixed both in https://github.com/nimblehq/ic-kmm-kayla-bruce/pull/82/commits/4c8105e96fe64c64ac35422a40d705014dd0326e

**Review by markgravity — APPROVED** [review]

**Review by thiennguyen0196 — APPROVED** [review]
LGTM 👏

**Pull request merged**